en/faqgeneral.md

```diff
@@ -154,3 +154,6 @@ A: See [How to improve the help page](faqcontributing/how-to-improve-the-help-pa
 
 A: This error message has been observed on systems that use [inotify](https://www.man7.org/linux/man-pages//man7/inotify.7.html). System calls to `inotify_init` and `inotify_add_watch` set `errno` to `EMFILE` when `inotify` has reached its limit. The most common reason is that `inotify` is running too many instances. To solve this problem, contact your system administrator and request that they increase the limits defined in `/proc/sys/fs/inotify/max_user_*` files.
 
+## Q: "I'm using JabRef with Linux (Gnome desktop version) and each time I want to see a pdf document in JabRef, the document viewer Evince is opened. I would prefer to open it with Okular."
```

Comment:
Thanks for the addition. We are currently trying to remove the FAQ, and incorporate it into the rest of the documentation. I thus would like to ask you if you could add this clarification in https://docs.jabref.org/setup/externalfiletypes (and maybe rework this section a bit).

Reply:
I don't think it fits in there as it's a system setting, independent of JabRef